lib/MusicBrainz/Server/Form/Admin/Attributes/Language.pm

```diff
@@ -1,6 +1,7 @@
 package MusicBrainz::Server::Form::Admin::Attributes::Language;
 
 use HTML::FormHandler::Moose;
+use MusicBrainz::Server::Translation qw( l );
```

Comment:
Not sure for what reason this line has been added and how is it related to MBS-10100.

Reply:
It's no longer needed since we changed validation, will remove.